Update log collection for Windows AMIs

KlwntSingh opened the pull request:
**Issue #, if available:**

**Description of changes:**
Docker is not installed for 1.24 AMIs, fix log collection for docker for 1.24 AMIs. Added log collection for containerd.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

<!-- If this is a security issue, please do not discuss on GitHub. Please report any suspected or confirmed security issues to AWS Security https://aws.amazon.com/security/vulnerability-reporting/ -->

**Testing Done**
I tested changes in script for EKS Windows Server 2019 Core AMIs for 1.20, 1.23 and 1.24 versions. 
1.20 have only docker installed, 1.23 have both docker and containerd installed, 1.24 have only containerd installed.
Here are steps I followed:
1. deployed pods on Windows Nodes.
2. RDP onto window node.
3. downloaded and executed script on node.
4. Script ran without any failures.
5. Verified both docker and containerd log files in output zip.

<!-- Include information regarding the testing that was completed with this changes. Where applicable, include details steps to replicate. -->

*[See this guide for recommended testing for PRs.](https://github.com/awslabs/amazon-eks-ami/blob/master/CONTRIBUTING.md#testing-changes) Some tests may not apply. Completing tests and providing additional validation steps are not required, but it is recommended and may reduce review time and time to merge.*

vsiddharth reviewed (commented):
Can you please add more details as to how this change was tested?

vsiddharth reviewed (approved):
LGTM but can you please squash the commits and update the testing section?

KlwntSingh commented:
> LGTM but can you please squash the commits and update the testing section?
Hi Sid, I squashed all commits and updated testing section with all steps I took for testing the changes.
